software-engineering-policies/SourceControl/RepositorySetupPolicy.md

```diff
@@ -0,0 +1,69 @@
+# Repository Setup Policy
+
+To ensure consistency in how our repositories are setup for access and automation, the following policy has been created. This focuses on the files within the codebase, the settings within the repo and automation we expect to run. 
+
+## Codebase setup
+
+So that we can ensure that anybody interacting with the codebase knows how they can perform certain actions, the following files should be added.
+
+### CODEOWNERS
```

Comment:
Obviuously you PR name is Github policies, however based on the file name/path I assume this applies to AzDo repos? If so AzDo uses a different approach to this.

Reply:
So this focuses on Github for the moment, however I am finding the AzDo way of doing it and will be updating/adding a new file to provide this information